easybuild/easyconfigs/i/iperf/iperf-2.1.9-GCCcore-10.2.0.eb

```
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-90.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'iperf'
+version = '2.1.9'
+
+homepage = 'http://iperf2.sourceforge.net/'
+description = """Iperf-2.1.9: TCP and UDP bandwidth performance measurement tool"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/iperf2/files', 'download')]
```

always use https if possible
```suggestion
source_urls = [('https://sourceforge.net/projects/iperf2/files', 'download')]
```